docker-compose.yml

```
@@ -0,0 +1,14 @@
+services:
```

version should be added at the beginning of the file
```suggestion
version: "3.7"

services:
```